ci/docker.sh

```diff
@@ -54,18 +53,15 @@ if [ "${is_a_release}" = "true" ]; then
   docker_tag_full="${github_ref_name}"
 fi
 
-# Building and publishing docker image
+# Load docker image
 if [ -n "${docker_tag_full:-}" ]; then
   if [ -n "${image_artifact:-}" ]; then
-    log_info "=== Using Docker image artifact from upstream 'zkVerify-qa' job ==="
-    log_info "Using GITHUB_WORKSPACE: ${GITHUB_WORKSPACE}"
+    log_info "=== Using Docker image artifact from upstream ==="
     image_name="$(docker load -i "${GITHUB_WORKSPACE}/${image_artifact}.tar" | awk '/Loaded image:/ { print $3 }')"
     log_info "=== Loaded image ${image_name} ==="
-    log_info "=== Renaming image ==="
-    docker tag "${image_name}" "${docker_hub_org}/${docker_image_build_name}:${docker_tag_full}"
   else 
-    log_info "=== Building Docker image: ${docker_hub_org}/${docker_image_build_name}:${docker_tag_full} ==="
-    docker build --build-arg PROFILE=production -f "${docker_file_path}" -t "${docker_hub_org}/${docker_image_build_name}:${docker_tag_full}" .
+    log_info "=== No artifact specified ==="
+    exit 1
```

Comment:
This should be `fn_die` vs `log_info` which will be in RED color and has exit 1 embedded. I would also add this smth about `--image-artifact` option for a better clarity of the error message